test/unit/parser.test.js

```diff
@@ -112,4 +112,18 @@ describe('Standard Markdown', () => {
       expect(output).toEqual("<p>C'est du texte avec <a href=\"http://example.com/\">un lien</a> et [une image][1]</p>\n");
     });
   });
+
+  describe('sanitize', () => {
+    it('will not change completely redacted input', () => {
+      const redacted = "C'est du texte avec [un lien][0] et [une image][1]";
+      const output = parser.sanitizeUnredacted(redacted);
+      expect(output).toEqual("C'est du texte avec [un lien][0] et [une image][1]\n");
+    });
+
+    it('will clear unredacted input', () => {
+      const unredacted = "This is some text with [a link](http://example.com/)";
+      const output = parser.sanitizeUnredacted(unredacted);
+      expect(output).toEqual('');
```

Comment:
For my understanding: why is this the empty string, instead of `'This is some text with '`?

Reply:
Design choice -- which I'm happy to change if we want the behavior you're suggesting! It's easier to detect unredactetd content if it's just cleared (and it will then default to English) but we could just remove the unredacted content. It wouldn't be a major change to this PR.